### PR TITLE
working starting point

### PR DESCRIPTION
**Pull request by pandark**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+## Install
+
+``` sh
+docker-compose up -d
+```
+
+### Add a (proxied) webapp
+
+Add the `A` and `AAA` entries in the authoritative DNS server, then run the
```
Comment (Exagone313) [inline README.md]: `AAAA`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+> under the terms of the GNU Affero General Public License as published by the
+> Free Software Foundation, either version 3 of the License, or (at your
+> option) any later version.
+> 
+> This program is distributed in the hope that it will be useful, but WITHOUT
+> ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+> FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+> more details.
+> 
+> You should have received a copy of the GNU General Public License along with
```
Comment (Exagone313) [inline README.md]: `Affero` missing

File: README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+> © 2019 Association Sans Nom and contributors (see CONTRIBUTORS.md)
+> 
+> This program is free software: you can redistribute it and/or modify it
+> under the terms of the GNU Affero General Public License as published by the
+> Free Software Foundation, either version 3 of the License, or (at your
+> option) any later version.
+> 
+> This program is distributed in the hope that it will be useful, but WITHOUT
+> ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+> FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
```
Comment (Exagone313) [inline README.md]: `Affero` missing

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    container_name: nginx-proxy
+    ports:
+      - "80:80"
+      - "443:443"
+    volumes:
+      - nginx-conf:/etc/nginx/conf.d:ro
+      - nginx-vhost:/etc/nginx/vhost.d:ro
+      - nginx-html:/usr/share/nginx/html:ro
+      - nginx-certs:/etc/nginx/certs:ro
+    #labels:
```
Comment (Exagone313) [inline docker-compose.yml]: What's the use for these comments?

Comment (pandark) [reply docker-compose.yml]: It doesn't work with the version of docker currently in Debian's repository, so for now we fall back to using `NGINX_PROXY_CONTAINER` and `NGINX_DOCKER_GEN_CONTAINER` environment variables in `letsencrypt-http` container.

**Pull request merged**